tests/unittests/api/helpers/test_files.py

```diff
@@ -111,6 +112,7 @@ def test_create_save_resized_image(self):
             upload_path = 'test'
             resized_image_url = create_save_resized_image(image_url_test, width, aspect_ratio, height, upload_path, ext='png')
             resized_image_file = app.config.get('BASE_DIR') + resized_image_url.split('/localhost')[1]
```

Comment:
The problem is actually here. This is assuming the localhost part of the URL to be static. You can split the URL using urlpase instead of this logic and have risk free path without regex replacement

```python
from urllib.parse import urlparse
o = urlparse('http://www.cwi.nl:80/%7Eguido/Python.html')
>> o.path
'/%7Eguido/Python.html'
```